components/CostsCard.vue

```diff
@@ -0,0 +1,290 @@
+<template>
+  <div class="card costs-card">
+    <!-- Todo: Make height dynamic. Matching header of time series. -->
+    <div class="card-header border-bottom-0 d-flex justify-content-between">
+      <div class="d-flex align-items-center">
+        <CIcon icon="cilChartPie" size="xl" class="mb-1 text-secondary" />
+        <h2 class="fs-5 m-0 ms-3 chart-header">
+          Losses
+        </h2>
+      </div>
+      <CostsLegend />
+    </div>
+    <div id="costsCardBody" ref="cardBody" class="card-body">
+      <div id="totalsContainer" ref="totalsContainer">
+        <h3 id="totalHeading" class="mt-0 mb-0 fs-6">
+          TOTAL
+        </h3>
+        <div id="gdpContainer" ref="gdpContainer" class="d-flex gap-1">
+          <p id="gdpTotalCostPercent" class="mt-0">
+            X.Y
+          </p>
+          <div class="align-self-end">
+            <p id="gdpTotalCostPercentageSymbol" class="mb-0">
+              %
+            </p>
+            <p id="gdpTotalCostPercentReferent" class="mt-0 mb-1">
+              of GDP
+            </p>
+          </div>
+        </div>
+        <div id="usdContainer" ref="usdContainer">
+          <div id="currency">
+            <p id="usdSymbol">
+              $
+            </p>
+            <p id="usdWord">
+              USD
+            </p>
+          </div>
+          <p id="totalCostPara" class="d-inline-block mb-0">
+            <span id="usdTotalCost">
+              <span>{{ totalCostAbbr?.amount }}</span>
+              <span id="totalCostUnit">
+                {{ totalCostAbbr?.unit }}
+              </span>
+            </span>
+          </p>
+          <p class="mt-0">
+            [insert note about GDP basis]
+          </p>
+        </div>
+      </div>
+      <CostsPie
+        v-if="pieStyle.height"
```

Comment:
Could 0 height happen? I tried making the window unreasonably tiny and the totals and pie just fell off the bottom of the card, but the pie was still shown. 
Might be worth setting a minimum height for the card?